NeuroID/src/main/java/com/neuroid/tracker/NeuroID.kt

```diff
@@ -646,15 +646,15 @@ private constructor(
     }
 
     @Synchronized
-    fun pauseCollection() {
+    fun pauseCollection(flushEvents: Boolean = true) {
```

Comment:
Do we want this to be public? The iOS repo made an internal version of this that allows the option of ForceSend or not, while the public method is always true

Reply:
can make this happen.